chia/full_node/block_store.py

```diff
@@ -10,6 +10,7 @@
 from chia.util.db_wrapper import DBWrapper
 from chia.util.ints import uint32
 from chia.util.lru_cache import LRUCache
+import zstd
```

Comment:
The imports are usually listed alphabetically in the following groups:

- stdlib
- third-party dependencies
- same-package dependencies

So you probably want this at line 5, after `import aiosqlite`

Reply:
it would be nice if `black` could do this :)
maybe we should just take the plunge and activate `isort` at some point.